stixcore/products/tests/test_housekeeping.py

```diff
@@ -60,13 +66,12 @@ def test_calibration_hk(levelb, idbm):
     hkl0 = MaxiReportL0.from_levelb(levelb)
     hkl1 = MaxiReportL1.from_level0(hkl0)
 
-    fits_procl1 = FitsL1Processor(Path(tempfile.gettempdir()))
+    fits_procl1 = FitsL1Processor(Path(tempfile.gettempdir()) / "stixcorefits")
```

Comment:
If using tmp_path you it will generate a unit dir path for each test

Reply:
thx nice fixture